pkg/util/wait/wait.go

```diff
@@ -60,6 +61,10 @@ func forHTTPStatusOk(ctx context.Context, cli SimpleHTTPClient, urltocheck strin
 					}
 				}
 			}
+			if _, ok := err.Err.(*x509.UnknownAuthorityError); ok {
```

Comment:
@jim-minter, was struggling to figure out this bit of code.  This would not fall into this section on my test script.  `x509.UnknownAuthorityError` is a struct and not specifically the error.

Is there a way to catch the x509?

Reply:
Thanks for the hints.